### PR TITLE
Making RazorPage.Write() handle TagHelperContent.

### PR DESCRIPTION
**Pull request by sornaks**

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
:shipit:

**Comment by sornaks** [comment]
Checked in. Thanks! https://github.com/aspnet/Mvc/commit/b581ff343c46a9479282ba900407fe5ada20a163

File: src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs [hunk 1]
```diff
@@ -298,7 +298,20 @@ public virtual void WriteTo(TextWriter writer, object value)
                     }
                     else
                     {
-                        WriteTo(writer, value.ToString());
+                        // This path is called when GetChildContentAsync() is called in tag helper
+                        // and content is not set.
```
Comment (dougbu) [inline src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs]: comment is specific to block starting a couple of lines down.  incorrect here.

**Comment by NTaylorMullen** [comment]
Nuke branch when you're ready.

**Pull request closed without merging**